src/main/webapp/admin_dashboard_style.css

```diff
@@ -182,7 +182,7 @@ main h1 {
   opacity: 1;
 }
 
-.popup {
+#popup {
```

Comment:
As I've mentioned above, it would be great if you adjust the size of the popup window to make it the same as the edit task pop up window on user profile page

Reply:
I believe we discussed compatability and uniformity would be a post-MVP consideration.